modules/rhel-adding-node.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-adding-node-{context}"]
+= Adding a RHEL compute machine to your cluster
+
+You can add compute machines that use Red Hat Enterprise Linux as the operating
+system to a {product-title} {product-version} cluster.
+
+.Prerequisites
+
+* You installed the required packages and performed the necessary configuration
+on the machine that you run the playbook on.
+* You prepared the RHEL hosts for installation.
+
+.Procedure
+
+Perform the following steps on the machine that you prepared to run the
+playbook:
+
+. Extract the pull secret for your cluster.
```

Comment:
s/cluster./cluster: 

And update the rest for consistency. We'll need to do this across the modules at some point anyway.